src/qtfirebaseadmob.h

```diff
@@ -404,14 +404,14 @@ class QtFirebaseAdMobInterstitialAdListener;
 class QtFirebaseAdMobInterstitial : public QtFirebaseAdMobBase
 {
     Q_OBJECT
+    Q_ENUMS(PresentationState)
```

Comment:
We *try to* follow Qt's style as much as possible. The Qt folks tend to place their ENUMS [under the declaration](https://github.com/qt/qtbase/blob/3fe74b76fd0eaf39d4c6681e2edca5adbf107883/src/corelib/io/qprocess.h#L114-L122)